src/main/resources/assets/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -149,10 +158,32 @@ local function tabulateCommon( bPaged, ... )
 end
 
 function tabulate( ... )
+    tArgs = { ... }
+    for i=1,#tArgs,2 do
```

Comment:
Well BombBloke you can emulate arg.n thing in later version of lua by simply going `local arg = table.pack(...)`. This causes exactly same functionality as arg did.